source/standards/accounts-with-third-parties.html.md.erb

```diff
@@ -1,143 +1,139 @@
+      G Suite access is revoked. However, you may still wish to clean
+      up the individual's account with the third-party service
 
-### Services that only provide individual accounts
+### Services only providing individual accounts
 
-For services which do not support organisation accounts, there is no
-choice but to share credentials.  This carries a number of problems:
+For services which don't support organisation accounts, there's no
+choice other than sharing sign-on credentials.
+
+This creates several risks:
 
 - It's not always easy to know who has access to a service,
-  particularly an occasionally-accessed service
-- If the last person with access leaves, we can lose access altogether
-- If the account is registered with an individual's email address, and
-  that person leaves, a number of features may be lost (such as
-  important notifications or "lost password" functionality)
-
-These recommendations try to reduce the risks of these adverse
-outcomes:
-
-- Create an account using a Google group email address rather than an
-  individual email address to avoid losing access to the account when
-  people leave (see notes below on using shared Google groups).
-- Store the password in your team's shared credential store.
-- If the account supports two-factor authentication, ensure each team
-  member has access to a second factor for the account
-- Ensure you have a documented process for removing individual's
-  access to the Google group and the credential store and for rotating
-  the credentials including any API keys and multi-factor
-  authentication.
-
-## Notes on using shared Google Groups
-
-Ensure that:
-
-- the group is able to receive emails from outside of the organisation by
+  particularly if it's accessed occasionally
+- If the last person with access leaves GDS, we can lose access to it altogether
+- If the account is registered with an individual's email address and
+  that person leaves a number of features may be lost, such as
+  important notifications or "lost password" functionality
+
+To reduce these risks, we recommend:
+
+- creating an account using a Google Group email address rather than an
+  individual email address, this avoids losing access to the account when
+  people leave (see [Shared Google Groups](#Shared Google Groups))
+- Storing passwords in your team's shared credential store
+- using two-factor authentication, if the account supports it, ensures each team
+  member has access to a second factor for their account
+- Ensuring you have a documented process for removing individual's access to:
+ - the Google Group
+ - credential store
+ - for rotating credentials including any API keys and multi-factor
+  authentication
+
+## Recommendations for specific third-party services
+
+There are some recommendations for specific third-party services you should consider.
+
+### Shared Google Groups
+{:#Shared Google Groups}
+
+When using shared Google Groups you should ensure:
```

Comment:
`Shared Google Groups` shouldn't be part of the `Recommendations for specific third-party services` section

Reply:
I moved the section into "Recommendations for specific third-party services" as it was specific to Google Groups, do you think it's important to merit it's own heading?